test/run-versions

```diff
@@ -65,15 +64,15 @@ testPython3_5() {
 
 testPython3_5_warn() {
   compile "python3_5_warn"
-  assertCaptured "python-3.5.6"
+  assertCaptured "Installing python-3.5.6"
   assertCaptured "security update!"
```

Comment:
I don't think the "security update" string appears anymore when installing an outdated version of 3.5, either.

Reply:
@vanastassiou The tests are passing on Travis, and this works for me in production:

```
$ git init
$ h create edmorley---buildpack https://github.com/heroku/heroku-buildpack-python
$ echo 'python-3.5.9' > runtime.txt
$ touch requirements.txt
$ git add -A && git commit -m '.' && git push heroku main
...
remote: -----> Python app detected
remote:  !     Python has released a security update! Please consider upgrading to python-3.5.10
remote:        Learn More: https://devcenter.heroku.com/articles/python-runtimes
remote: -----> Installing python-3.5.9
remote: -----> Installing pip 20.1.1, setuptools 47.1.1 and wheel 0.34.2
```

If you are running the tests for this repository locally, could you try cloning https://github.com/heroku/heroku-buildpack-python to a new directory to rule out modifications to the existing clone?

Or if you are seeing this in production, please could you file a support ticket at https://help.heroku.com